### PR TITLE
bpo-39019: Implement missing __class_getitem__ for http.cookies.Morsel

### PR DESCRIPTION
**Pull request by isidentical**
https://bugs.python.org/issue39019

### CODE REVIEW

File: Misc/NEWS.d/next/Library/2019-12-10-21-27-37.bpo-39019.7o43Gq.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Implement ``__class_getitem__`` for ``http.cookies.Morsel`.
```
Comment (pablogsal) [inline Misc/NEWS.d/next/Library/2019-12-10-21-27-37.bpo-39019.7o43Gq.rst]: The test are failing because you have unbalanced quotes in http.cookies.Morsel

Comment (isidentical) [reply Misc/NEWS.d/next/Library/2019-12-10-21-27-37.bpo-39019.7o43Gq.rst]: Thanks for the catch Pablo.

File: Misc/NEWS.d/next/Library/2019-12-10-21-27-37.bpo-39019.7o43Gq.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Implement ``__class_getitem__`` for ``http.cookies.Morsel``.
```
Comment (ilevkivskyi) [inline Misc/NEWS.d/next/Library/2019-12-10-21-27-37.bpo-39019.7o43Gq.rst]:
```suggestion
Implement dummy ``__class_getitem__`` for ``http.cookies.Morsel``.
```

**Comment by ilevkivskyi** [comment]
@isidentical please don't force-push, because the reviewers will not get a notification from GitHub.

@pablogsal I think you can look at this again.

**Comment by isidentical** [comment]
I see, thanks for advice @ilevkivskyi

File: Lib/http/cookies.py [hunk 3]
```diff
@@ -293,6 +293,17 @@ def __init__(self):
         for key in self._reserved:
             dict.__setitem__(self, key, "")
 
+    def __class_getitem__(cls, type):
+        """Provide minimal support for using this class as generic
+        (for example in type annotations).
+
+        See PEP 484 and PEP 560 for more details. For example, `Morsel[T]`
+        is a valid expression at runtime (type argument `T` indicates the
+        type used for mode). Note, no type checking happens at runtime, but
```
Comment (ilevkivskyi) [inline Lib/http/cookies.py]: I think this is wrong. This type should represent value type for a key-value pair held. But probably it is best if someone familiar with this code will take a look.

**Comment by isidentical** [comment]
Closing since pep 585 accepted

**Pull request closed without merging**